llama_finetuning.py

```diff
@@ -24,6 +24,7 @@
     BitsAndBytesConfig
 )
 import torch.distributed as dist
+from optimum.bettertransformer import BetterTransformer
```

Comment:
Please remove from here as we have this in the try-catch-block for optional install of optimum library